app/locale/en_US/Potoky_ImageAutoImport.csv

```diff
@@ -0,0 +1,22 @@
+"Import of Images", "Импорт Изображений"
```

Comment:
This file should contain translate to English. In our case the same values as the keys. Add extra locale folder (ru_RU) for Russian translation and put there this file